services/types.go

```diff
@@ -12,4 +14,5 @@ type OnboardingTicket struct {
 	ExpirationDate    int64                 `json:"expirationDate,string"`
 	SubjectRole       OnboardingSubjectRole `json:"subjectRole"`
 	StorageQuotaInGiB *uint                 `json:"storageQuotaInGiB,omitempty"`
+	StorageClusterUID types.UID             `json:"storageClusterUID"`
```

Comment:
Only capital the first latter of an abbreviations
```suggestion
	StorageClusterUid types.UID             `json:"storageClusterUid"`
```